internal/actions/publishEvent.go

```diff
@@ -138,6 +147,44 @@ func processNPM(cfg *config.Config, event *shared.CliEvent, path string, version
 	return nil
 }
 
+func processGo(cfg *config.Config, event *shared.CliEvent, path string, version string) error {
+	lang := "go"
+	if cfg.Config == nil {
+		return fmt.Errorf("empty config for %s language target in directory %s", lang, path)
+	}
+
+	langCfg, ok := cfg.Config.Languages[lang]
+	if !ok {
+		return fmt.Errorf("no %s config in directory %s", lang, path)
```

Comment:
These error messages seem backwards

Reply:
yes thanks

Reply:
What no actually. No typescript config in the directory blah